common-arcana.lic

```diff
@@ -133,6 +142,28 @@ module DRCA
     DRC.retreat(settings.ignored_npcs) unless spell['skip_retreat']
   end
 
+  def prepare_to_cast_runestone(spell, settings)
```

Comment:
This should be named `prepare_to_cast_runestone?` since it is a predicate

Reply:
How will that change the function?

Reply:
http://ruby-for-beginners.rubymonstas.org/objects/predicates.html

Reply:
Added question mark. Testing.